### PR TITLE
Add action points

### PR DESCRIPTION
**Pull request by Benjamin-Lee**
**Did you add yourself as a [contributor](https://github.com/Benjamin-Lee/deep-rules/blob/master/contributors.md) if this is your first contribution?**

- [x] Yes, I added myself or am already a contributor

**Any more details?**
In response to https://github.com/Benjamin-Lee/deep-rules/issues/174#issuecomment-484719672.

### CODE REVIEW

File: content/04.baselines.md [hunk 1]
```diff
@@ -15,3 +15,5 @@ It can be tempting to train baseline models with these conventional methods usin
 Hu and Greene [@doi:10.1101/385534] discuss a "Continental Breakfast Included" effect by which unequal hyperparameter tuning for different learning algorithms skews the evaluation of these methods, especially when the performance of an algorithm varies substantially with modest changes to its hyperparameters.
 Those wishing to compare different learning algorithms should tune the settings of both traditional and DL-based methods to optimize performance before making claims about relative performance differences.
 The performance comparison among DL models and many other ML approaches is informative only when the models are similarly well-tuned.
+
+To sum this tip up, create and fully tune several traditional models such as logistic regressors or random forests before implementing a DL model.
```
Comment (rasbt) [inline content/04.baselines.md]: I think you meant "regression models" instead of "regressors" (in stats, regressors refers to what is called "inputs features" in ML)

**Comment by rasbt** [comment]
About these action points ...

Is this meant to meet some requirement by the journal (i.e., some stylistic field that they want as page header or 1 sentence conclusion) or is this more of an optional summary of the paragraphs that we want to provide? (Just asking to get a better idea of what the limitations are here)

**Comment by Benjamin-Lee** [comment]
Not a requirement of the journal, just trying to make it easier to implement the paper's recommendations. I know that one of the issues that has been raised (can't remember where) was that the article is too abstract, so this was meant to make it a little more concrete.

**Comment by rasbt** [comment]
I see. Thanks for the info!

File: content/09.overfitting.md [hunk 2]
```diff
@@ -19,3 +19,5 @@ If the model performs no better on real data than randomly scrambled data, then
 Additionally, one must be sure that their data are not skewed or biased, such as by having confounding and scientifically irrelevant variables that the model can pick up on [@arxiv:1807.00431].
 In this case, simply holding out test data is insufficient.
 The best remedy for confounding variables is to [know your data](#know-your-problem) and to test your model on truly independent data.
+
+In essence, split your data into training, tuning, and testing sets to assess the performance of the model on data it truly has not seen before, while also being cognizant of the danger of skewed or biased data artificially inflating accuracy.
```
Comment (agitter) [inline content/09.overfitting.md]: I'd focus on not using the test set multiple times in the actionable summary.  That is a major problem in many manuscripts.

File: content/11.interpretation.md [hunk 3]
```diff
@@ -10,11 +10,10 @@ While we know that accurately predicting an outcome doesn't imply learning the c
...
 This rule contradicts medical understanding - having asthma doesn't make pneumonia better!
 This rule was supported by the data (pneumonia patients with a history of pneumonia tended to receive more aggressive care), so the neural network also learned to make predictions according to this rule.
 Guiding treatment decisions according to the predictions of the neural network would have been disastrous, even though the neural network had high predictive accuracy.
 
 To trust deep learning models, we must combine knowledge of the training data ([Tip 4](#know-your-problem)) with inspection of the model ([Tip 8](#blackbox)).
-By probing data domains where the model succeeds and contrasting with domains where the model fails, we can identify the internal logic and deduce scientific conclusions.
-In this way, we can move beyond fitting predictive models toward building understanding.
+To move beyond fitting predictive models toward building understanding and deduce scientific conclusions, probe data domains where your model succeeds and contrast them with domains where your model fails in order to identify your model's internal logic.
```
Comment (agitter) [inline content/11.interpretation.md]: The "internal logic" was there before, but that phrase could risk teaching readers to over-interpret their models as cautioned against in the first paragraph.

File: content/09.overfitting.md [hunk 4]
```diff
@@ -19,3 +19,5 @@ If the model performs no better on real data than randomly scrambled data, then
 Additionally, one must be sure that their data are not skewed or biased, such as by having confounding and scientifically irrelevant variables that the model can pick up on [@arxiv:1807.00431].
 In this case, simply holding out test data is insufficient.
 The best remedy for confounding variables is to [know your data](#know-your-problem) and to test your model on truly independent data.
+
+In essence, split your data into training, tuning, and single-use testing sets to assess the performance of the model on data it truly has not seen before. Additionally, be cognizant of the danger of skewed or biased data artificially inflating accuracy.
```
Comment (agitter) [inline content/09.overfitting.md]: Newline between sentences
